File: jserver-core/src/test/java/com/github/gr3gdev/jserver/ServerTest.java
package com.github.gr3gdev.jserver;

import org.junit.Test;

import java.io.IOException;

public class ServerTest {

    @Test
    public void testStartAndStop() throws IOException, InterruptedException {
        final Server server = new Server();
        for (int i = 0; i < 10; i++) {
            server.start();
            Thread.sleep(100);
            server.stop();
        }
    }

}

File: jserver-core/src/test/kotlin/com/github/gr3gdev/jserver/ServerTest.kt
package com.github.gr3gdev.jserver

import org.junit.Test

class ServerTest {

    @Test
    fun testStartAndStop() {
        val server = JServer.server()
        for (i in 0..10) {
            server.start()
            Thread.sleep(100)
            server.stop()
        }
    }

}